styles.css

```diff
@@ -65,22 +76,19 @@ h1 {
   line-height: 100%;
 }
 
-#paragraph {
-  height: 126px;
-  margin: 20px 0 0;
-  align-self: center;
+.paragraph {
+  margin-top: 20px;
   font-family: 'Poppins', sans-serif;
-  font-style: bold;
   font-size: 14px;
   line-height: 21px;
   color: #1a2236;
   letter-spacing: -0.01em;
 }
```

Comment:
- [ OPTIONAL]You did an excellent job on this project, but it would be much more appealing if you matched your font-weight to the figma design; please alter this to fit the Figma design. :wink: 

- [ OPTIONAL] Also change the name of the author given on the template design and put your name, please change this because it may cause misunderstanding when recruiters approach you. :wink: 

Your Design :point_down: 
![great1](https://user-images.githubusercontent.com/88894525/194714699-837f12ed-6dc4-49f1-9773-b714f9ee2d67.png)

Figma Design

![greet2](https://user-images.githubusercontent.com/88894525/194714757-51001f4b-d3f8-4064-aa70-044d521d114d.png)